### ISSUE TITLE
[BUG] NPE at core.gui.theme.ThemeManager.getAvailableThemeNames

### ISSUE THREAD

**Issue by akasolace** [akasolace/HO#647]
**Describe the bug**

**To Reproduce**
File -> Option

**Screenshots**
![image](https://user-images.githubusercontent.com/1136496/88568915-f75cf880-d039-11ea-8a72-bc8923820c97.png)

**Comment by akasolace** [comment]
@tychobrailleur is it a bug you experience also ?

**Comment by tychobrailleur** [comment]
> @tychobrailleur is it a bug you experience also ?

Nope, it isn't.  The `FarbPanel` has been removed in the #639 PR to solve a similar issue though.

**Comment by akasolace** [comment]
ah ok so maybe once this PR is merged and I rebase to master the issue might be gone ... would be nice because I need to access the preference panel to continue work on the update functionality and I don't feel looking at another issue at the moment :-)

**Comment by akasolace** [comment]
closed by #639
